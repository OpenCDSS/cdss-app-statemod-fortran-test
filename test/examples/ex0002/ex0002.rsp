# _______________________________________________________
#
# ex0002
# _______________________________________________________
#
# StateMod Response File
#
# Type                                    Name
# ______________________________________  ________________________________________________________________________
Control                                 = ex0002.ctl
River_Network                           = ex0002.rin
Reservoir_Station                       = ex0002.res
Diversion_Station                       = ex0002.dds
StreamGage_Station                      = ex0002.ris
Instreamflow_Station                    = ex0002.ifs
Instreamflow_Right                      = ex0002.ifr
Reservoir_Right                         = ex0002.rer
Diversion_Right                         = ex0002.ddr
Operational_Right                       = ex0002.opr
Evaporation_Monthly                     = ex0002.eva
Stream_Base_Monthly                     = ex0002.rim
Diversion_Demand_Monthly                = ex0002.ddm
Instreamflow_Demand_AverageMonthly      = ex0002.ifa
Reservoir_Target_Monthly                = ex0002.tar
Reservoir_Historic_Monthly              = ex0002.eom
StreamGage_Historic_Monthly             = ex0002.rih
Diversion_Historic_Monthly              = ex0002.ddh
GeographicInformation                   = ex0002.gis
OutputRequest                           = ex0002.out
DelayTable_Monthly                      = ex0002.dly
