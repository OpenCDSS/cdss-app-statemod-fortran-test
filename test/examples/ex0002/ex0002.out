# Example 2.9
# *.xou; Output Request file for StateMod Example
#
# Type (e.g. Diversion, StreamGage or Reservoir, or All)
# 
All
#
# Parameter (e.g. TotalSupply, SimEOM, RiverOutflow, or All)
# 
All
#
# ID Name Type and Print Code (0=no, 1=yes);
# Note: id = All prints all
#       id = -999 = stop
#       default is to turn on all stream gages (FLO)
#
All



Dem_3        Irrigation Demand _3     DIV     1
Dem_2        Irrigation Demand _2     DIV     1
Dem_1        Municipal Demand _1      DIV     1
Isf_1        Instream Demand          ISF     1
Res_1        Reservoir_1              RES     1
Riv_10       Exist. Div. & Inflow     FLO     1
Riv_30       Inflow                   FLO     1
Riv_80       USGS Gauge               FLO     1
Riv_20       Intermediate River Node  OTH     0
Riv_50       Conf. Mainstem & Trib A  OTH     0
Opr_1        Opr Target Release       OPR     1
-999        
